src/gt4py/storage/allocators.py

```diff
@@ -330,12 +356,12 @@ def tensorize(
 
 device_allocators[core_defs.DeviceType.CPU] = NumPyLikeArrayBufferAllocator(
     device_type=core_defs.DeviceType.CPU,
-    array_ns_ref=np,
+    array_ns_ref=cast(_NumPyLikeNamespace, np),
```

Comment:
But doing this I make it `_NumPyLikeNamespace[Any]`, right? But I don't see that a more specific one makes sense, because of the reason above: it should be both array interface and dlpack, no?

Reply:
Yes, you're right. Either we remove the typevar from the NumPyLikeNamespace  or we parametrize it with the actual array class